scripts/profiler/processed_events.py

```diff
@@ -32,24 +32,24 @@ def match_event_processing(self):
             'event_processing_start')
         self.event_processing_end_id = self.raw_data.get_event_type_id(
             'event_processing_end')
-
-        if (self.event_processing_start_id is None) or \
-          (self.event_processing_end_id is None):
-            self.tracking_execution = False
-            for ev in self.raw_data.events:
-                self.tracked_events.append(TrackedEvent(ev, None, None))
-            return
-
-        for i in range(1, len(self.raw_data.events)):
+        for i in range(0, len(self.raw_data.events)):
```

Comment:
`range(len(self.raw_data.events))`

If possible, shorter is better, fewer arguments is better, shorter comments are better. Smaller chance for mistakes :)